package.json

```diff
@@ -98,7 +98,7 @@
     "semver": "^6.0.0",
     "state-switch": "^0.6.2",
     "watchdog": "^0.8.1",
-    "wechaty-puppet": "^0.15.25",
+    "wechaty-puppet": "^0.15.26",
```

Comment:
Do you want to upgrade this version as well?

Reply:
Yes, I should have.

Bumped the version to `0.15.27`